pipelines/pingcap/tidb/release-6.5/pod-pull_common_test.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
```

Comment:
go version is go1.20，please update to hub.pingcap.net/jenkins/centos7_golang-1.20:latest